backend/src/main/scala/cromwell/backend/BackendJobExecutionActor.scala

```diff
@@ -28,7 +28,7 @@ object BackendJobExecutionActor {
 /**
   * Workflow-level actor for executing, recovering and aborting jobs.
   */
-trait BackendJobExecutionActor extends BackendLifecycleActor with ActorLogging {
+trait BackendJobExecutionActor extends BackendJobLifecycleActor with ActorLogging {
```

Comment:
It makes it more friendly on the implementation side because you don't have to pass around a jobDescriptor to every method you write although you know for a fact that your actor is scoped with this job already